ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -589,177 +672,96 @@ def test_repeat(
 
 
 @st.composite
-def _tile_helper(draw):
-    dtype, value, shape = draw(
-        helpers.dtype_and_values(
-            available_dtypes=ivy_np.valid_dtypes, ret_shape=True, min_num_dims=1
-        )
-    )
-    reps = draw(
-        helpers.dtype_and_values(
-            available_dtypes=(ivy_np.int8, ivy_np.int16, ivy_np.int32, ivy_np.int64),
-            shape=(len(shape),),
-            min_value=0,
-            max_value=10,
+def _split_helper(draw):
+    noss_is_int = draw(
```

Comment:
add a comment explaining this

Reply:
Explained what is happening with `_split_helper` in a comment